content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Istio 1.9.0
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `istioctl install` will detect different Istio version installed (istioctl, control plan version) and display warning.
+  ([Issue #18487](https://github.com/istio/istio/issues/18487))
+
+- **Added** post-install/in-place upgrade verification of control plane health. Use `--verify` flag with `istioctl install` or `istioctl upgrade`  ([Issue #21715](https://github.com/istio/istio/issues/21715))
+
+- **Added** Add pprof endpoint to pilot-agent.
+  ([Issue #28040](https://github.com/istio/istio/issues/28040))
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow user specify whether the istio crds will be installed. ([Pull Request #28346](https://github.com/istio/istio/pull/28346))
+
+- **Added** Allow enabling gRPC logging with --log_output_level for pilot.
+  ([Issue #28482](https://github.com/istio/istio/issues/28482))
+
+- **Added** `istioctl apply` as an alias for `istioctl install`.
+  ([Issue #28753](https://github.com/istio/istio/issues/28753))
+
+- **Added** `--browser` flag to `istioctl dashboard`, which controls whether you want to open a browser to view the dashboard.
```

Comment:
Does this have any user impact?  My istioctl dashboard cmd always open up a browser today.

Reply:
```go
dashboardCmd.PersistentFlags().BoolVar(&browser, "browser", true, 
 	"When --browser is supplied as false, istioctl dashboard will not open the browser. "+ 
 		"Default is true which means istioctl dashboard will always open a browser to view the dashboard.") 
```